Assingmenty/DiceRollGame/RandomNumberProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿internal class RandomNumberProvider
+{
+    public static int Generate()
+    {
+        Random rnd = new Random();
```

Comment:
uważam że takie skróty są złe. nic nie dają tak naprawdę, a zawsze są mniej czytelne niż pełne słowo. Wywodzą się chyba z czasów jak rozmiar source codu miał znaczenie, ale dziś nie ma.
Also, 7 to magic number. Zrobiłabym consta na DiceSidesNumber równego 6, i tutaj wołała między 1 a DiceSidesNumber + 1

Reply:
ja tez, ale jak zajrzalam w doumentacje microsoftu to tak tam bylo, wiec pomyslalłam ze to moze standard, jak np używanie aliasu pd na Pandas w Pythonie

Reply:
Booooże. W tej dokumentacji to są takie babole że o ja pierdolę, nie mówiąc że czasem sobie przeczą sami z tym co zalecają a z tym przed czym przestrzegają -.-